packages/clay-button/webpack.config.js

```diff
@@ -26,7 +26,7 @@ module.exports = {
 	},
 	plugins: [new webpack.optimize.ModuleConcatenationPlugin()],
 	resolve: {
-		mainFields: ['esnext:main', 'main'],
+		mainFields: ['esnext:main', 'main', 'jsnext:main'],
```

Comment:
Hey @matuzalemsteles, do we really need the `jsnext:main` main field here? I would simply keep the `esnext:main` if that's going to be the default future one. All the other ones can go. Same for the other `config` files.

Reply:
Hey @jbalsas,

I did some testing, we do not really need the `jsnext:main` field in `config` files. But we'll have to leave the `main` field because of the dependencies of some packages.

I'll send the change.